portal/views.py

```diff
@@ -130,10 +138,14 @@ def get(self, request):
                 ).data,
             }
             history_list.append(context)
-        for entry in history_list:
-            entry["poll_statistics"] = get_demographic_breakdown(entry["poll"]["id"])
         return Response(sorted(history_list, key=lambda i: i["poll"]["expire_date"], reverse=True))
 
+    @action(detail=False, methods=["get"])
+    def recent(self, request, pk=None):
+        recent_poll_vote = PollVote.objects.filter(user=request.user).latest("created_date")
+        recent_poll = recent_poll_vote.poll
+        return Response(RetrievePollSerializer(recent_poll).data)
```

Comment:
I think this needs to be a `RetrievePollVoteSerializer`. On the `list` function it is a bit funky but I'm using that `RetrievePollVoteSerializer`, and then calling `RetrievePollSerializer`, but then converting it into the form of `RetrievePollVoteSerializer` for consistency